container_accessibility/tests/test_user_limit.py

```diff
@@ -0,0 +1,20 @@
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
+from odoo.tools import config
+
+
+class TestUserLimit(TransactionCase):
+    def test_exceeded(self):
+        config["user_limit"] = "5"
+        with self.assertRaises(UserError):
+            for i in range(10):
+                self.env["res.users"].create(
+                    {
+                        "name": "User %s" % i,
+                        "login": "User %s" % i,
+                        "groups_id": [
+                            self.ref("container_accessibility.group_restricted")
+                        ],
+                    }
+                )
+        config["user_limit"] = "0"
```

Comment:
Would this affect the actual config file? I know its a strange question but would like to know more about it :) If yes, are we setting the user_limit to 0 by default?

Reply:
No just in the current worker, by default it's 0 so I just put it back to the default value here so other tests where users are created don't fail.